terraform/boards/variables.tf

```diff
@@ -15,3 +15,9 @@ variable "kafka_stats_dataset" {
   type        = string
   default     = "ExampleKafkaStatsDataset"
 }
+
+variable "refinery_metrics_dataset" {
+  description = "Dataset to use for refinery metrics"
+  type        = string
+  default     = "ExampleRefineryMetricsDataset"
```

Comment:
We should make this the default. Which is "[Refinery Metrics](https://github.com/honeycombio/refinery/blob/14f5c7f00e923372bdbc094d0b685ab5be654339/config_complete.toml#L299)"

Reply:
Done!